collector/container.go

```diff
@@ -139,6 +139,14 @@ func (c *ContainerMetricsCollector) Collect(ch chan<- prometheus.Metric) error {
 	return nil
 }
 
+// containerClose closes the container resource
+func containerClose(c hcsshim.Container) {
+	err := c.Close()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
We do not want to quit the exporter if this happens, so I'd say a `log.Error` is sufficient.

Reply:
Okay. changed to log.Error.